source/compiler/aslresources.y

```diff
@@ -666,6 +667,21 @@ PinFunctionTerm
         error PARSEOP_CLOSE_PAREN   {$$ = AslDoError(); yyclearin;}
     ;
 
+ClockInputTerm
```

Comment:
I'm wondering if the (mis)location is deliberate (to split Pin*() resources instead of putting as in other places).

Reply:
No, it isn't. I don't think ClockInput has to break Pin* resources. I can tidy it up

Reply:
Please submit a new PR if and when you tidy it up! Also please do a fresh git pull/fetch to ensure you have the latest ACPICA since there might have been some new changes in some of those files since the last stable release dated 03312023! Thanks a lot for the contributions!